src/init.cpp

```diff
@@ -441,6 +441,7 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-checkmempool=<n>", strprintf("Run checks every <n> transactions (default: %u)", defaultChainParams->DefaultConsistencyChecks()));
         strUsage += HelpMessageOpt("-checkpoints", strprintf("Disable expensive verification for known chain history (default: %u)", DEFAULT_CHECKPOINTS_ENABLED));
         strUsage += HelpMessageOpt("-disablesafemode", strprintf("Disable safemode, override a real safe mode event (default: %u)", DEFAULT_DISABLE_SAFEMODE));
+        strUsage += HelpMessageOpt("-enablerpcmethod", "Enables a deprecated RPC method");
```

Comment:
I think `enablerpcmethod` is already clear, especially since this is a hidden option and the help text explains that it's used for enabling a deprecated method.

I don't think there's validation for any of our arguments.